hbase-server/src/test/java/org/apache/hadoop/hbase/security/token/TestZKSecretWatcher.java

```diff
@@ -140,35 +141,35 @@ public void testKeyUpdate() throws Exception {
     // sanity check
     assertTrue(KEY_MASTER.isMaster());
     assertFalse(KEY_SLAVE.isMaster());
-    int maxKeyId = 0;
 
     KEY_MASTER.rollCurrentKey();
     AuthenticationKey key1 = KEY_MASTER.getCurrentKey();
     assertNotNull(key1);
-    LOG.debug("Master current key: "+key1.getKeyId());
+    LOG.debug("Master current key (key1) {}", key1);
 
     // wait for slave to update
     Thread.sleep(1000);
     AuthenticationKey slaveCurrent = KEY_SLAVE.getCurrentKey();
     assertNotNull(slaveCurrent);
     assertEquals(key1, slaveCurrent);
-    LOG.debug("Slave current key: "+slaveCurrent.getKeyId());
+    LOG.debug("Slave current key (key1) {}", slaveCurrent);
 
     // generate two more keys then expire the original
     KEY_MASTER.rollCurrentKey();
     AuthenticationKey key2 = KEY_MASTER.getCurrentKey();
-    LOG.debug("Master new current key: "+key2.getKeyId());
+    LOG.debug("Master new current key (key2) {}", key2);
     KEY_MASTER.rollCurrentKey();
     AuthenticationKey key3 = KEY_MASTER.getCurrentKey();
-    LOG.debug("Master new current key: "+key3.getKeyId());
+    LOG.debug("Master new current key (key3) {}", key3);
 
     // force expire the original key
-    key1.setExpiration(EnvironmentEdgeManager.currentTime() - 1000);
+    key1.setExpiration(EnvironmentEdgeManager.currentTime() - 100000);
```

Comment:
I know this is a backport, but mind explaining why change this value? I'm not familiar with the original improvement...

Reply:
I'm not sure of the original intention here as the -1000 should also make the key expired but it doesn't make any difference if the key expired long time ago.
If you prefer I can revert back this line.

https://github.com/apache/hbase/blob/branch-2.2/hbase-server/src/main/java/org/apache/hadoop/hbase/security/token/AuthenticationTokenSecretManager.java#L238-L256

Reply:
As you said this is a backport, what is the value on master branch?

Reply:
There is no difference on the master branch which would require the 100000 expiration.